app/services/-private/session.ts

```diff
@@ -0,0 +1,8 @@
+
+import SessionService from 'ember-simple-auth/services/session';
+
+export default class OsfSession extends SessionService {
+    handleAuthentication() {
```

Comment:
I suspect we can get away with rewriting [this line](https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/app/decorators/check-auth.ts#L52) to use `session.handleAuthentication(transition.targetName)` or something instead, so we can remove the `ember-simple-auth.events.session-service` deprecation handler. That may allow us to not have to stub the session service in our tests either?